client/src/views/Settings/Tabs/EditProfile/index.vue

```diff
@@ -1,140 +1,251 @@
+            :rules="[checkURL]"
+            lazy-rules
+          >
+            <template v-slot:prepend>
+              <q-icon class="fab fa-youtube youtube" />
+            </template>
+          </q-input>
+          <q-input
+            label="Instagram"
+            placeholder="Add Instagram link to your profile"
+            v-model="profile.socialAccounts.instagram"
+            :rules="[checkURL]"
+            lazy-rules
+          >
+            <template v-slot:prepend>
+              <q-icon class="fab fa-instagram instagram" />
+            </template>
+          </q-input>
+          <q-input
+            label="Facebook"
+            placeholder="Add Facebook link to your profile"
+            v-model="profile.socialAccounts.facebook"
+            :rules="[checkURL]"
+            lazy-rules
+          >
+            <template v-slot:prepend>
+              <q-icon class="fab fa-facebook-f facebook" />
+            </template>
+          </q-input>
+        </q-card-section>
+      </q-card>
+
       <div class="q-pt-md">
         <q-btn
           label="Edit profile"
           type="submit"
-          color="primary"
-          no-caps
+          rounded
+          color="secondary"
         />
         <q-btn
           label="Reset"
           type="reset"
-          color="primary"
+          color="secondary"
           flat
           class="q-ml-sm"
-          no-caps
         />
       </div>
+
     </q-form>
   </q-page>
 </template>
 
 <script>
 import Avatar from '@/components/Avatar.vue'
+import EditImage from './EditImage.vue'
 import {
   requiredField,
   checkNicknameField,
-  checkUserDescriptionField
+  checkUserDescriptionField,
+  checkURL
 } from '@/utilities/validation.js'
-const suggestedInterests = ['nature', 'IT', 'innovation', 'tourism', 'music', 'football', 'flowers']
+import authService from '@/services/auth.js'
 
 export default {
   components: {
-    Avatar
+    Avatar,
+    EditImage
   },
 
   data () {
+    const emptyProfile = {
+      nickname: '',
+      bio: '',
+      status: '',
+      avatar_url: null,
+      user_background: null,
+      socialAccounts: {
+        youtube: '',
+        instagram: '',
+        facebook: ''
+      }
+    }
     return {
-      filterOptions: suggestedInterests,
       photoUrl: '',
       backgroundUrl: '',
-      profile: {
-        nickname: '',
-        bio: '',
-        status: '',
-        avatar_url: null,
-        user_background: null,
-        interests: null,
-        socialAccounts: {
-          youtube: '',
-          instagram: '',
-          facebook: ''
-        }
+      profile: { ...emptyProfile },
+      emptyProfile,
+      notifyParameters: {
+        textColor: 'white',
+        actions: [{ icon: 'close', color: 'white' }],
+        timeout: 3000
       }
     }
   },
 
   computed: {
     profileGetter () {
-      return this.$store.getters.profileGetter
+      return this.$store.getters['profile/myProfile']
```

Comment:
Please use getters like this -  https://vuex.vuejs.org/ru/guide/getters.html#%D0%B2%D1%81%D0%BF%D0%BE%D0%BC%D0%BE%D0%B3%D0%B0%D1%82%D0%B5n%D1%8C%D0%BD%D0%B0%D1%8F-%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D1%8F-mapgetters